covid_moonshot_ml/datasets/utils.py

```diff
@@ -136,6 +139,33 @@ def cdd_to_schema(cdd_csv, out_json, achiral=False):
     pic50_vals = [float(c.strip("<> ")) for c in df[pic50_key]]
     df["pIC50"] = pic50_vals
     df["pIC50_range"] = pic50_range
+    semiquant = df["pIC50_range"].astype(bool)
+
+    ci_lower_key = (
+        "ProteaseAssay_Fluorescence_Dose-Response_Weizmann: IC50 "
+        "CI (Lower) (µM)"
+    )
+    ci_upper_key = (
+        "ProteaseAssay_Fluorescence_Dose-Response_Weizmann: IC50 "
+        "CI (Upper) (µM)"
+    )
+    ## Calculate 95% CI in pIC50 units based on IC50 vals (not sure if the
+    ##  difference should be taken before or after taking the -log10)
```

Comment:
@jchodera do you know the answer to this?

Reply:
`-log(x+y) != -log(x) + -log(y)`
`-log(x) + -log(y) = -log(x*y)`

Therefore you should add the +/- confidence intervals to the calculated value *before* taking the log.

I think that's right?

Reply:
sorry for the confusion, I think I meant to delete this comment at some point. I believe this is answered here:
https://choderalab.slack.com/archives/C03B535P36C/p1657215011957369

Reply:
That's right---you want to convert each IC50 separately, then use that pIC50 confidence interval as desired.